decoimpact/business/entities/rules/classification_rule.py

```diff
@@ -93,16 +91,22 @@ def execute(
                     comparison_val = read_str_comparison(criteria, "<")
                     comparison = data < float(comparison_val)
 
+                elif criteria_class == "larger_equal":
+                    comparison_val = read_str_comparison(criteria, ">=")
+                    comparison = data >= float(comparison_val)
+
+                elif criteria_class == "smaller_equal":
+                    comparison_val = read_str_comparison(criteria, "<=")
+                    comparison = data <= float(comparison_val)
+
```

Comment:
we do not have an else condition. Is that problematic? Should we add a catch for that?

Reply:
These are the only options, provided by "type_of_classification" provided by string_parser_utils.py. If there is an exception it will be catched there already. So adding an else in this classification_rule file will resolve in unreachable code